react/week4/my-app/src/index.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+import FindUser from "./FindUser";
+import context from "./context";
+
+const ContextProvider = context.Provider;
+
+class App extends React.Component {
+  state = { userName: "", usersList: [], isLoading: false };
+  handelUserSearch = this.handelUserSearch.bind(this);
+
+  handelUserSearch(e) {
+    const value = e.target.value;
+    this.setState({ userName: value });
+
+    const URL = `https://api.github.com/search/users?q=${value}`;
+    fetch(URL)
+      .then((data) => data.json())
+      .then((response) => {
+        const items = response.items;
+
+        this.setState({
+          usersList: items.map((item) => {
+            return item;
```

Comment:
I don't think you need to map here! Think about what `map` does and consider if it's necessary to map here, only to return the element without modification.